internal/kafka/test/integration/kafkas_test.go

```diff
@@ -2396,6 +2397,93 @@ func TestKafka_RemovingExpiredKafkas_NoStandardInstances(t *testing.T) {
 	Expect(kafkaDeletionErr).NotTo(HaveOccurred(), "Error waiting for kafka deletion: %v", kafkaDeletionErr)
 }
 
+func TestKafka_ReassignKafkaClusterFull(t *testing.T) {
```

Comment:
Nice. later on once we've the re-assignment logic we could check that the cluster is assigned to a different cluster

This  test case is better suited to be under this file https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/97efc10fe4cde1952558f18148b40c15411117a7/internal/kafka/test/integration/data_plane_endpoints_test.go